plugin/zoom/API/JWTClient.php

```diff
@@ -0,0 +1,380 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+namespace Chamilo\PluginBundle\Zoom;
+
+use Exception;
+use Firebase\JWT\JWT;
+
+/**
+ * Class JWTClient.
+ *
+ * @see https://marketplace.zoom.us/docs/api-reference/zoom-api
+ * @package Chamilo\PluginBundle\Zoom
+ */
+class JWTClient
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class JWTClient has 14 non-getter- and setter-methods. Consider refactoring JWTClient to keep number of methods under 10.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5625225)